### DISCUSSION TITLE
Xiaomi Robot Vacuum Mop 2S

### DISCUSSION THREAD

**Discussion by rucanunes** [rytilahti/python-miio#1629] [General]
Hello,

Is it possible to integrate the Xiaomi Robot Vacuum Mop 2S?

Thank you!

**Comment by rytilahti** [comment]
You could try using `miiocli genericmiot` on the current git master to see if that works. This device might be supported (with some extra features) using a separate integration, but if the device speaks miot you will have at least the basic controls.

**Reply by benedikt-wue** [reply]
any news about the Vacuum Mop 2S?
